packages/webapp/src/browser/modules/docusign/Docusign.component.tsx

```diff
@@ -0,0 +1,228 @@
+                    {envelope.title}
+                </StyledHeading>
+                <StyledDate>
+                    {t('docusign:created_on')}
+                    {formatDate(envelope.createdDate)}
+                </StyledDate>
+            </Main>
+            <Secondary>
+                <Lozenge
+                    variant={statusConfig.variant}
+                    subtle
+                    icon={statusConfig.icon}
+                >
+                    {t(`docusign:status${envelope.status.charAt(0).toUpperCase() + envelope.status.slice(1)}`)}
+                </Lozenge>
+                <Link external>
+                    {t('docusign:link')}
+                </Link>
+            </Secondary>
+            {envelope.reasonText && (
+                <Bottom>
+                    <Reason>
+                        <strong>
+                            {t('docusign:reason_label')}
+                            &nbsp;:&nbsp;
+                        </strong>
+                        {envelope.reasonText}
+                    </Reason>
+                </Bottom>
+            )}
+        </Envelope>
+    );
+});
+
+// Main DocusignPage Component
+export const DocusignPage: FunctionComponent = memo(() => {
+    const { t, i18n } = useTranslation();
+
+    const formatDate = useCallback(
+        (date: Date): string => new Intl.DateTimeFormat(i18n.language, {
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+        }).format(date),
+        [i18n.language],
+    );
+
+    const envelopeData = useMemo(() => [
+        {
+            id: uuid(),
+            title: 'TFSA account opening',
+            createdDate: new Date(),
+            status: 'send' as const,
+            reasonText: 'This document requires your signature to finalize the account opening.',
+        },
+        {
+            id: uuid(),
+            title: 'Lorem ipsum dolor sit amet',
+            createdDate: new Date(),
+            status: 'completed' as const,
+            reasonText: 'Consectetur adipiscing elit, sed do eiusmod tempor incididunt.',
+        },
+        {
+            id: uuid(),
+            title: 'Lorem ipsum dolor sit amet',
+            createdDate: new Date(),
+            status: 'declined' as const,
+            reasonText: 'Consectetur adipiscing elit, sed do eiusmod tempor incididunt.',
+        },
+    ], []);
+
+    const renderEnvelopes = useCallback(
+        () => envelopeData.map((envelope) => (
+            <EnvelopeItem
+                key={envelope.id}
+                envelope={envelope}
+                t={t}
+                formatDate={formatDate}
+            />
+        )),
+        [envelopeData, t, formatDate],
+    );
+
+    const count = useMemo(() => envelopeData.length, [envelopeData]);
+
+    const accordionItems = useMemo(() => [
+        {
+            title: t('docusign:accordion_title', { count }) ?? `DocuSign envelope status (${count})`,
```

Comment:
Et rendu là, comme ton hook va dépendre de `envelopeData`
```suggestion
            title: t('docusign:accordion_title', { count: envelopeData.length }) ?? `DocuSign envelope status (${count})`,
```